scripts/linters/pylint_extensions.py

```diff
@@ -165,6 +176,30 @@ def visit_call(self, node):
                             func_name))
                     num_positional_args_unused -= 1
 
+        for arg in keyword_args:
+            # If there is *args and **kwargs in the function definition skip the
+            # check because we can use keywords arguments in function call even
```

Comment:
Do we want developers to do this i.e, use keyword arg in place of **kwargs? If yes then it's okay to skip this else we should not skip these cases. DO you see this pattern in the codebase anywhere?

Reply:
Yes, it is used in many places, see example below https://github.com/oppia/oppia/blob/61c354ed294946775e49d65813b2c8fada4d5ca7/scripts/linters/pre_commit_linter.py#L82

Reply:
@Hudda, I had a discussion with @bansalnitish on this, can you please work with him to discuss this and get it resolved?

Reply:
I've discussed this with @bansalnitish and I agree that this is a problem. But we have a problem in implementation. Let's see an example:

```
def function(arg1, *args, **kwargs)

caller function(arg1='x', a='1')
```
by parsing calling method we get keyword_arguments = [arg1, a].
But from the function call, we will get keyword_arguments = []
now we do not have a way to identify which one is a keyword argument and which one is not.

Reply:
@Hudda, Thanks for the explanation! You can follow these steps:
1. Considering keyword_arguments are generated in the visit_call, I'm assuming you have checked how these values are generated, if not maybe you can check and see if we can do something?
2. Try implementing it (ask mentor for help if required) if you still see it's not possible, create an issue and leave a TODO comment!

Reply:
@Hudda can you please take this up?

Reply:
Added the todo comment.